backend/development.ini.sample

```diff
@@ -153,22 +170,19 @@ preview_cache_dir = %(basic_setup.preview_cache_dir)s
 ; preview.jpg.allowed_dims = 256x256
 # examples with multiples values:
 # preview.jpg.allowed_dims = 256x256,512x512,1024x1024
-## Preview dimensions can be set as restricted. If set as restricted, access
+## Preview dimensions can be set as restricted. If set as True, access
 ## endpoint to get any other preview dimensions than allowed_dims will
 ## return error
 ; preview.jpg.restricted_dims = False
-# preview.jpg.restricted_dims = True
 
 ### Session ###
-# shortcut for pyramid_beaker specific config
-# see tracim_web app section to see where those shortcut are used
-advanced_setup.session_data_dir = %(basic_setup.sessions_data_root_dir)s/sessions_data
-advanced_setup.session_lock_dir = %(basic_setup.sessions_data_root_dir)s/sessions_lock
 
 ### pyramid_beaker parameters ###
 session.type = file
-session.data_dir = %(advanced_setup.session_data_dir)s
-session.lock_dir = %(advanced_setup.session_lock_dir)s
+; session.data_dir = %(here)s/sessions_data
+session.data_dir = %(basic_setup.sessions_data_root_dir)s/sessions_data
+; session.lock_dir = %(here)s/sessions_data
+session.lock_dir = %(basic_setup.sessions_data_root_dir)s/sessions_lock
```

Comment:
Why do not comment them and let default value ?

Reply:
That the decision we have choosen to clarify that default value is different
see solution 1 of https://github.com/tracim/tracim/issues/2925

Reply:
solution 2 is possible but more complicated. We will check this another time.